packages/orbit-components/src/InputGroup/InputGroup.stories.tsx

```diff
@@ -155,16 +158,11 @@ const selectOptionsError = [
   { value: "November", label: "November" },
   { value: "December", label: "December" },
 ];
-export const Error = ({ label, flex, error, help, selectValue }) => {
-  return (
-    <InputGroup
-      label={label}
-      flex={flex}
-      onChange={action("onChange")}
-      onFocus={action("onFocus")}
-      onBlur={action("onBlur")}
-    >
-      <InputField placeholder="DD" error={error} help={help} />
+
+export const Error: Story = {
+  render: ({ help, selectValue, ...args }) => (
+    <InputGroup {...args}>
+      <InputField placeholder="DD" help={help} error="Something went wrong on day field" />
```

Comment:
I would actually make the `error` here controllable, so we can change the text or even remove it to display the error on the Select.

Same as the [current](https://kiwicom.github.io/orbit/?path=/story/inputgroup--error&args=error:) master.

Reply:
Fixed ✅